bliss/datasets/galsim_galaxies.py

```diff
@@ -104,21 +104,21 @@ def test_dataloader(self):
 class SDSSGalaxies(pl.LightningDataModule, Dataset):
     def __init__(
         self,
-        num_workers=0,
-        batch_size=64,
-        n_batches=10,
-        n_bands=1,
-        slen=53,
-        min_flux=1e3,
-        max_flux=3.5e5,
-        min_a_d=0.8,
-        max_a_d=6.5,
-        min_a_b=0.8,
-        max_a_b=3.6,
-        background=865.0,
-        pixel_scale=0.396,  # SDSS
+        num_workers,
+        batch_size,
+        n_batches,
+        n_bands,
+        slen,
+        min_flux,
+        max_flux,
+        min_a_d,
+        max_a_d,
+        min_a_b,
+        max_a_b,
+        background,
+        pixel_scale,  # SDSS
+        flux_sample,
         psf_image_file: str = None,
```

Comment:
I think `psf_image_file` having a `None` default value is a bug; I believe this function will raise an error if it's not a valid file. We should change this to not have a default value. 

@ismael-mendoza

Reply:
Yes I agree with this. I believe the default should be `data/psField-000094-1-0012-PSF-image.npy`